openlcs/products/models.py

```diff
@@ -75,6 +75,8 @@ def create_component(self, data):
 
     def add_components_from_nvrs(self, nvrs, type="SRPM", arch="src"):
         release_node, _ = self.get_or_create_release_node()
+        # Not add container nvr as "SRPM" component.
+        nvrs = [nvr for nvr in nvrs if 'container-source' not in nvr]
```

Comment:
This is issue 2 from https://issues.redhat.com/browse/OLCS-230
![two-components](https://user-images.githubusercontent.com/16874024/189366814-7435cdd8-ae95-4ee4-9d07-c01dcddd3c85.png)

when importing a source container, in our code, first will create a component with SRPM type, it is incorrect.
so here exclude `source container`
for the binary container, because we don't support it now, if we need to support it, I think we should move this logic to another workflow, not at first import.

Reply:
Updated, remove this create srpm component step, if we add this step here, should check nvr belongs with type, and each component will be created in `send_package_data` step